src/Covid19Dashboard/PlotChart/index.jsx

```diff
@@ -339,11 +365,13 @@ PlotChart.propTypes = {
 
 PlotChart.defaultProps = {
   plots: [],
-  xTitle: null,
-  yTitle: null,
+  xTitle: undefined,
+  yTitle: undefined,
+  axisLabelMaxLength: undefined,
+  axisLabelFontSize: undefined,
```

Comment:
this default value will overwrites the default value for `labelFontSize` props in `PlotChartAxisTick` if no `axisLabelFontSize` props has been passed into this component? let's keep them consistant?